src/reducers/entities/posts.js

```diff
@@ -3,7 +3,7 @@
 
 import {PostTypes, SearchTypes, UserTypes} from 'action_types';
 import {Posts} from 'constants';
-import {comparePosts} from 'utils/post_utils';
+import {comparePosts, combineSystemPost} from 'utils/post_utils';
```

Comment:
This method should be `combineSystemPosts` so that it's clearer that it does all posts at once